examples/example_taggged_signals.rs

```diff
@@ -80,9 +83,11 @@ impl actix::Handler<woab::Signal<usize>> for WindowActor {
                 None
             }
             "remove_addend" => {
-                if let Some((widgets, _)) = self.addends.remove(msg.tag()) {
-                    self.widgets.lst_addition.remove(&widgets);
-                    self.recalculate();
+                if let Some((addend, _)) = self.addends.remove(msg.tag()) {
+                    let lst_addition = self.widgets.lst_addition.clone();
+                    woab::schedule_outside(move || {
```

Comment:
@piegamesde - this is the style. What do you think?

I don't love having to `clone()` things that need to be sent outside with the closure, but at least this is a burden Rust coders should be familiar with and know how to handle. And I don't really see a way around it.

Reply:
Cloning is a pattern so common to plain Rust gtk, that there is a convenient macro for it. I didn't miss it after switching to WoAB (although I still used once or twice), but I don't really mind it either.